### PR TITLE
Update README with usage docs/examples

### PR DESCRIPTION
**Pull request by mikeastock**
Copying information from website to README

### CODE REVIEW

**Comment by PascalTurbo** [comment]
Thank you for that PR - great work.
I wouldn't merge it until checking if every explained example really works. The website seems to be a bit outdated and I found one type (/lib/feedzira) only by reading.
If you have the time to check them - do it. Otherwise I will do it in a few days.

File: README.md [hunk 1]
```diff
@@ -24,3 +142,49 @@ Feedjira.configure do |config|
...
+
+* [Solve for All][solve]: Solve for All combines search engine and feed parsing
+  while protecting your privacy. It's even extendable by the community!
+
+[Feedbin]: https://feedbin.com/
+[Stringer]: https://github.com/swanson/stringer
+[BlogFeeder]: https://apps.shopify.com/blogfeeder
+[Feedbunch]: https://github.com/amatriain/feedbunch
+[old]: http://theoldreader.com/
+[solve]: https://solveforall.com/
```
Comment (HParker) [inline README.md]: Nice! cool to see friends of feedjira right in the readme. :)

**Comment by mikeastock** [comment]
Fixed broken examples in the README

**Comment by mikeastock** [comment]
I realized right after I merged this that @PascalTurbo proposed #363. I believe this PR is an example of self merging being acceptable. I would love to know how everyone else feels.

**Pull request merged**